charts/immich/templates/machine-learning/deployment.yaml

```diff
@@ -3,37 +3,44 @@ apiVersion: apps/v1
 metadata:
   name: {{ template "common.fullname" . }}-machine-learning
   labels:
-    app.kubernetes.io/name: {{ template "common.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/component: "immich-machine-learning"
+    app.kubernetes.io/component: "machine-learning"
     {{- include "common.labels" . | nindent 4 }}
 spec:
   revisionHistoryLimit: 0
   replicas: {{ .Values.server.replicas }}
```

Comment:
Shouldnt this be .Values.machinelearning.replicas?

Reply:
You didnt change it but we might as well clean it up :)

Reply:
🤦‍♀️